packages/cli/test/dev/integration-1.test.ts

```diff
@@ -90,6 +92,65 @@ test(
   })
 );
 
+test('[vercel dev] only logs request/response for functions', async () => {
```

Comment:
Nit pick: add this to `integration-5.test.ts` instead since that is currently the fastest

Reply:
Good call. [`5703795` (#9411)](https://github.com/vercel/vercel/pull/9411/commits/570379572c58762be8e363f0149a1f73b3421c8d)